### ISSUE TITLE
"Packages" is missing

### ISSUE THREAD

**Issue by hansgschossmann** [OPCFoundation/UA-EdgeTranslator#4]
The README.md says in "Installation": UA Edge Translator is available as a pre-built Docker container and will run on any Docker- or Kubernetes-enabled edge device. See "Packages" in this repo for details.

I have not found anything with "Packages". Did i missed anything?

**Comment by barnstee** [comment]
Yes: https://github.com/OPCFoundation/UA-EdgeTranslator/pkgs/container/ua-edgetranslator
:-)
